### ISSUE TITLE
Unable to compile with gcc 10 | Arch Linux solution = downgrade

### ISSUE THREAD

**Issue by Studentenfutter** [bmschmidt/wordVectors#61]
If installing the package in Arch Linux, it failes to compile with the following error:

```
ccache gcc -shared -L/usr/lib64/R/lib -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -o wordVectors.so tmcn_word2vec.o word2phrase.o -L/usr/lib64/R/lib -lR
/usr/bin/ld: word2phrase.o:(.bss+0x18): multiple definition of `vocab'; tmcn_word2vec.o:(.bss+0x78): first defined here
collect2: Fehler: ld gab 1 als Ende-Status zurück
make: *** [/usr/share/R//make/shlib.mk:6: wordVectors.so] Fehler 1
ERROR: compilation failed for package ‘wordVectors’
```
This seems to be an error raised by gcc 10 and should be fixable quite easily.

### Temporary Fix (see [here](https://github.com/polybar/polybar/issues/2097))

Downgrade gcc and related packages (in my case gcc-libs and gcc-fortran) **simultaneously** to version `9.3.0-1`

Either downgrade if you still have the old cached version or download them from
https://archive.archlinux.org/packages/g/gcc/gcc-9.3.0-1-x86_64.pkg.tar.zst
https://archive.archlinux.org/packages/g/gcc-libs/gcc-libs-9.3.0-1-x86_64.pkg.tar.zst and 
https://archive.archlinux.org/packages/g/gcc-fortran/gcc-fortran-9.3.0-1-x86_64.pkg.tar.zst

Install using `sudo pacman -U gcc-9.3.0-1-x86_64.pkg.tar.zst gcc-libs-9.3.0-1-x86_64.pkg.tar.zst gcc-fortran-9.3.0-1-x86_64.pkg.tar.zst`. It's important to downgrade them at the same time because one depends on the other.

To prevent automated updates off gcc till the issue is resolved add the three packages to `IgnorePkg` in `/etc/pacman.conf`

**Comment by maslinych** [comment]
Build with gcc 10 fixed in PR #63

**Comment by bmschmidt** [comment]
Accepting @maslinych's pull and closing.
